main.go

```diff
@@ -12,6 +12,7 @@ import (
 	"os"
 	"time"
 
+	"github.com/go-logr/logr"
```

Comment:
I'm trying to see why this change occurred,  but I cant spot anything, did we bring in the wrong logger somewhere?

Reply:
I used https://pkg.go.dev/github.com/go-logr/logr#NewContext in main.go to get a Context from a logger. But I revisited and learned it's not necessary.